__tests__/app.test.js

```diff
@@ -41,6 +41,49 @@ describe("1. GET/api/topics", () => {
 });
 
 /***articles***/
+describe("8. GET/api/articles", () => {
+  test("Status:200, responds with an articles array of aritcle objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
```

Comment:
Double check the ticket requirements. We don't want a body here. If you imagine it as a blog or news site, this is like the list of titles we can click :)